spack/package.py

```diff
@@ -91,8 +91,8 @@ class Octopus(AutotoolsPackage, CudaPackage):
     depends_on("libxc@2:2", when="@:5")
     depends_on("libxc@2:3", when="@6:7")
     depends_on("libxc@2:4", when="@8:9")
-    depends_on("libxc@5.1.0:", when="@10:")
-    depends_on("libxc@5.1.0:", when="@develop")
+    depends_on("libxc@5.1.0:5", when="@10:")
+    depends_on("libxc@5.1.0:5", when="@develop")
     with when("+mpi"):  # list all the parallel dependencies
```

Comment:
Allow `libxc@6` for most recent versions (which?) of Octopus.

Something like `depends_on("libxc@5.1.0:", when="@12:")`